packages/amplify-provider-awscloudformation/src/utils/remove-dependent-function.ts

```diff
@@ -0,0 +1,39 @@
+import { appsyncTableSuffix } from 'amplify-category-function/lib/provider-utils/awscloudformation/utils/constants';
+import { $TSContext, $TSObject, pathManager } from 'amplify-cli-core';
+import path from 'path';
+import * as TransformPackage from 'graphql-transformer-core';
+import { updateDependentFunctionsCfn } from 'amplify-category-function';
```

Comment:
Don't create a dependency between the provider plugin and a category plugin, use the invokePluginMethod function in context to invoke a function from a category plugin that supplied to you by the plugin platform. Same way we do for auth.

Also this PR missed to add a package.json entry for the category plugin (which we should not need because of the above).